woocommerce/woocommerce-ios
# Create new View for the list of Card Readers

Part of: #7193 

This PR consolidates the different "Card Reader Manual" links into a unique row in the IPP Settings screen, as part of the IPP manual consolidation project (ref: p91TBi-7SX-p2)

### Description
- Modifies the In-Person Payments screen under the `.consolidatedCardReaderManuals `feature flag
- Adds a new Card Readers View
- Adds images for each reader
- Implements `SafariView` for all reader manuals through a `NavigationLink`
- Extracts the view model into `CardReadersViewViewModel`

| Before | After | Flow
|---|---|---|
| <img width="350" src="https://user-images.githubusercontent.com/3812076/177267195-22aace46-f320-4806-b183-86f4b404bfc7.png"> | <img width="350" src="https://user-images.githubusercontent.com/3812076/177256852-3d1a4131-e7cc-4d21-803b-d6d0587ce248.png"> | ![RMC 2022-07-05 V1 done](https://user-images.githubusercontent.com/3812076/177265495-f5a9ffa3-8adf-4399-84cc-aff06b4ead25.gif)

### Testing instructions
- Go to Settings > In-Person Payments, see how the individual rows for each manual have disappeared in favor of a new `Card Reader manuals` row
- Tap on `Card Reader manuals` and see 3 options:
- See how each option redirects to the online documentation for that specific manual


---
